tests/README.md

```diff
@@ -0,0 +1,69 @@
+# Liquid Haskell Tests
+
+## Directory Structure
+
+- `./tests.cabal`: master cabal file for the tests, containing tests organized
+  by group.
+- `./harness/`: The test driver itself.
+- `./app/Main.hs`: A dummy Main file shared between test group executables.
+- `./benchmarks/`: benchmark-style tests, which are run for judging the
+  performance of Liquid.
+- `./errors/`: Tests for checking specific error messages. For the list of
+  messages expected, see `./harness/Test/Groups.hs`.
+- everything else in `./`: folders for different classes of tests, often
+  containing `pos` and `neg` subfolders for positive and negative tests
+  respectively.
+
+## `test-driver` Executable
+
+See the code for comments and documentation that is likely more up to date than
+this file. The test driver parses the output of stack or cabal to summarize the
+result of compiling specific test groups, represented in `tests.cabal` as
+separate executables. The rest of this file describes how to modify the test
+suite by adding new tests.
+
+### Adding a Test
+
+Adding a new test to an existing test group is straight-forward: create a new
```

Comment:
You are welcome to pick your style. But if I were writing these instructions, I would avoid making judgements about how easy or difficult the procedures are. The reader will make her own opinion, I reckon.

Reply:
Changed.